pallets/credit/src/lib.rs

```diff
@@ -819,7 +731,11 @@ pub mod pallet {
         ) -> DispatchResult {
             ensure!(price != 0u32.into(), Error::<T>::PriceZero);
             let who = ensure_signed(origin)?;
-            ensure!(Self::is_admin(&who), Error::<T>::NotAdmin);
+            let credit_data = Self::user_credit(&who).unwrap();
```

Comment:
`Self::get_credit_score(&who).unwrap_or(0)`
not using unwrap() since some non-credit user also can call this funciton